config/samples/component-filtering.yaml

```diff
@@ -0,0 +1,11 @@
+apiVersion: core.spinoperator.dev/v1alpha1
+kind: SpinApp
+metadata:
+  name: hello-salutation-spinapp
+spec:
+  image: "ghcr.io/kate-goldenring/spin-operator/examples/spin-salutations:20241022-144454"
```

Comment:
A TODO should be left or issue opened to come back and point at the spinkube sample app image once this merges.